lib/hushed/documents/request/hash_converter.rb

```diff
@@ -42,6 +42,10 @@ def bill_to_hash
           def normalize_sku(sku)
             sku.chomp("-SET")
           end
+
+          def state(address)
+            address.state ? address.state.name : address.state_name
```

Comment:
I believe I did this very change in another shipping implementation that utilizes XML! Eerily familiar. Anyway, looks 👍